e2e/testcases/death/helpers.ts

```diff
@@ -30,11 +35,11 @@ export type DeathDeclarationInput = {
 const declaration = {
   deceased: {
     name: {
-      firstNames: faker.name.firstName('male'),
-      familyName: faker.name.lastName('male')
+      firstNames: faker.name.firstName('male') + generateRandomSuffix(),
+      familyName: faker.name.lastName('male') + generateRandomSuffix()
     },
     gender: 'male',
-    birthDate: getRandomDate(75, 200),
+    age: random(20, 100),
```

Comment:
Let's revisit this if there are too many clashes. Then we just need to keep track of the values